### DISCUSSION TITLE
Which cases studies need data to list in the SCI data project?

### DISCUSSION THREAD

**Discussion by mrchrisadams** [Green-Software-Foundation/software_carbon_intensity_data#9] [Q&A]
This is a thread to start collating the the data that people need when trying to record a SCI for a service.

We know there a number of SCI case studies underway, but I'm not aware of them all, nor which data is currently in use.

Can we try collating them here? Given there datasets are not too common, before reaching for some complicated data CMS like you see in action on https://data.climatesubak.org, it might be wiser to either create a "data" section on the awesome green software project, or have a simple page linking to datasets here first.

@jawache , I'm tagging you as I'm not sure if Daniele, who was helping project manage this SCI data work is on Github.

Below is a list of all the case studies in action right now. A useful starting step might be to see which data being described as missing in each case so we have a summary here of the data we have, and do not have.

https://github.com/Green-Software-Foundation/software_carbon_intensity/issues?q=is%3Aissue+is%3Aopen+label%3A%22Case-study+submission%22

**Comment by DanniBradu** [comment]
@mrchrisadams here are the requests from Shrini for his case study:

1) Power consumption value for RAMS – DDR2, DDR3, DDR4.
 
Some references links below point to approx. 1-4 Watts for 4 GB -16 GB RAM
https://www.google.co.in/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&cad=rja&uact=8&ved=2ahUKEwj6y4GZsvj2AhVCyzgGHduGC_IQFnoECAMQAw&url=https%3A%2F%2Fwww.tomshardware.com%2Freviews%2Fintel-core-i7-5960x-haswell-e-cpu%2C3918-13.html&usg=AOvVaw1G1i-G-RB59wKrHrhi-k7b
 
https://medium.com/teads-engineering/estimating-aws-ec2-instances-power-consumption-c9745e347959
 
Regarding memory consumption, we found some interesting data from DRAM manufacturers. Micron provides a calculator [7] and a power model [8] giving theoretical power consumption for each command for DDR4 memory systems.
In their example, we can see a power consumption of ~0,41W/GB. In its official documentation, memory manufacturer Crucial [9] says that: “As a rule of thumb, however, you want to allocate around 3 watts of power for every 8GB of DDR3 or DDR4 memory”. Which equals to ~0,38W/GB.
 
2) Manufacturing emissions cost for these RAMS . In the spreadsheet I see that we have for 128 GB RAM it is 150 KgCO2eq. Can we pro rate this for lower RAM sizes?
 
3) Networking emissions of g/GB. – We need to define this based on the architecture of the application. From the application analysis we will have the Data in and Data out of the specific application. What we would need as reference is the g/GB and we could do it in different ways
- for web application
- for mobile apps
